cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -11,21 +11,27 @@ damping_method_enum = gen.enum([
                        gen.const("truncation",          int_t, 2, "truncation")],
                      "enum types for the damping_method")
 
-gen.add("damping_method", int_t, 0, "The damping method to use (edited via an enum)", 0, 0, 2, edit_method=damping_method_enum)
+constraints_enum = gen.enum([ 
```

Comment:
should we call this wln_constraints_enum as we lateron might get additional constraints when using additional tasks (extra lines for Jacobian) or constraints (using Nullspace projection)?